core/src/Xios.cpp

```diff
@@ -82,7 +81,7 @@ void Xios::finalize()
  */
 void Xios::configure()
 {
-    // check if xios is enabled in the nextsim configuration
+    // Check if XIOS is enabled in the neXtSIM-DG configuration
```

Comment:
I've been trying to standardize the capitalization to nextSIM or nextSIM-DG

Reply:
Addressed in 2dbc23a8.